src/async-wrap.cc

```diff
@@ -207,25 +209,23 @@ Local<Value> AsyncWrap::MakeCallback(const Local<Function> cb,
   }
 
   if (ran_init_callback() && !pre_fn.IsEmpty()) {
-    try_catch.SetVerbose(false);
```

Comment:
I think not setting verbose to `false` here means that errors thrown from with async pre-hooks would be forwarded to the active domain's error handler. Do you have time to verify that theory? If that's the case, I'm not sure that makes sense, and what the right thing to do is.

Setting verbose to `false` also means that the debugger wouldn't catch any exception in async pre-hooks, so it might not be ideal too.